DependencyInjection/App/Presenters/AnotherPresenter.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+
+namespace App\Presenters;
+
+use App;
+use Vendor;
+
+class AnotherPresenter extends Vendor\Presenters\SomePresenter
+{
+	/**
+	 * @var \Vendor\Services\AnotherVendorService
```

Comment:
Očividně nastavené, protože mi to nedělá. Zřejmě volba `Editor`, `Intentions`, `PHP` a `Import class`. Případně dej najít nastavení `fully`. Jinak nevím, kde by se to mohlo měnit